spec/twiml/messaging_response_spec.rb

```diff
@@ -3,93 +3,185 @@
 describe Twilio::TwiML::MessagingResponse do
   context 'Testing Response' do
     it 'should allow empty response' do
-      r = Twilio::TwiML::MessagingResponse.new
-      expect(r.to_s).to eq('<?xml version="1.0" encoding="UTF-8"?><Response/>')
+      expected_doc = parse <<-XML
+        <Response/>
+      XML
+      response = Twilio::TwiML::MessagingResponse.new.to_s
+      doc = parse(response)
+      expect(doc).to be_equivalent_to(expected_doc).respecting_element_order
     end
 
     it 'should allow using to_xml instead of to_s' do
-      r = Twilio::TwiML::MessagingResponse.new
-      expect(r.to_xml).to eq('<?xml version="1.0" encoding="UTF-8"?><Response/>')
+      expected_doc = parse <<-XML
+        <Response/>
+      XML
+      response = Twilio::TwiML::MessagingResponse.new.to_xml
+      doc = parse(response)
+      expect(doc).to be_equivalent_to(expected_doc).respecting_element_order
     end
 
     it 'should allow populated response' do
-      r = Twilio::TwiML::MessagingResponse.new
-      r.message(body: 'Hello')
-      r.redirect('example.com')
-
-      expect(r.to_s).to eq('<?xml version="1.0" encoding="UTF-8"?><Response><Message>Hello</Message><Redirect>example.com</Redirect></Response>')
+      expected_doc = parse <<-XML
+        <Response>
+          <Message>Hello</Message>
+          <Redirect>example.com</Redirect>
+        </Response>
+      XML
+      response = Twilio::TwiML::MessagingResponse.new
+      response.message(body: 'Hello')
+      response.redirect('example.com')
+      doc = parse(response)
```

Comment:
do we not need to call `to_s` on the response first?

Reply:
The [`parse` method](https://github.com/twilio/twilio-ruby/pull/386/files#diff-93830fa29d616f7c87903d08b5b1b29aR36) calls `to_s` to save on repetition.

Reply:
gotcha